corehq/apps/receiverwrapper/views.py

```diff
@@ -160,7 +160,11 @@ def _submission_error(request, message, count_metric, metric_tags,
     return response
 
 
-def _record_metrics(tags, submission_type, response, timer=None):
+def _record_metrics(tags, submission_type, response, timer=None, xform=None):
+    if xform and xform.metadata:
+        lag = xform.received_on - xform.metadata.timeEnd
```

Comment:
This is causing a large number of submission 500s (at least on production): https://sentry.io/organizations/dimagi/issues/1114102925/?project=136860&query=is%3Aunresolved&statsPeriod=14d, https://app.datadoghq.com/dashboard/4fg-n5r-sgn/500-errors?from_ts=1563156475297&live=true&tile_size=m&to_ts=1563329275297&fullscreen_widget=2943715324358578&fullscreen_section=overview

Reply:
I think this should fix it: https://github.com/dimagi/commcare-hq/pull/24866